src/pages/basic_data/app_admin/central_import/CentralImportRegistration.tsx

```diff
@@ -0,0 +1,70 @@
+import { Link as RouterLink, useParams } from 'react-router-dom';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { Box, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import { createRegistrationFromImportCandidate, fetchImportCandidate } from '../../../../api/registrationApi';
+import { getRegistrationLandingPagePath, RegistrationParams } from '../../../../utils/urlPaths';
+import { setNotification } from '../../../../redux/notificationSlice';
+import { getIdentifierFromId } from '../../../../utils/general-helpers';
+
+export const CentralImportRegistration = () => {
+  const { t } = useTranslation();
+  const { identifier } = useParams<RegistrationParams>();
+  const dispatch = useDispatch();
+
+  const importCandidateQuery = useQuery({
+    queryKey: ['importCandidate', identifier],
+    queryFn: () => fetchImportCandidate(identifier),
+    meta: { errorMessage: t('feedback.error.get_registrations') },
+  });
+
+  const activateRedirectButton = () => {
+    setIsActive(false);
+  };
+
+  const importCandidateMutation = useMutation({
+    mutationFn: importCandidateQuery.data
+      ? () => createRegistrationFromImportCandidate(importCandidateQuery.data)
+      : undefined,
+    onSuccess: () => {
+      dispatch(
+        setNotification({
+          message: t('feedback.success.create_registration'),
+          variant: 'success',
+        })
+      );
+      activateRedirectButton();
```

Comment:
Trenger egentlig ikke å opprette en ny funksjon for dette da vi kan bruke funksjonen vi allerede har fra `useState` i stedet:
```suggestion
      setIsActive(false);
```
Merk at deklareringen av `useState` da må flyttes litt opp, slik at den kommer før bruken.

(Ser ut som det har blitt litt kluss med navngiving her også, da `activateRedirectButton` setter `active` til `true`, så det er egentlig en `disableRedirectButton`)